packages/web-react/src/components/Modal/README.md

```diff
@@ -100,6 +100,12 @@ dialog shrinks to fit the height of its content (if smaller than the viewport).
 <ModalDialog isExpandedOnMobile>…</ModalDialog>
 ```
 
+#### ⚠️ DEPRECATION NOTICE
+
+The `isExpandedOnMobile` prop will be set to `true` by default in the next major release and the ModalDialog will be
+expanded on mobile by default. It will be possible to re-collapse the inside by setting the `isExpandedOnMobile` to
```

Comment:
```suggestion
expanded on mobile by default. It will be possible to re-collapse the inside by setting the `isExpandedOnMobile` prop to `false` value.
```